pigeon_tracker/camera_parameters.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import yaml
+
+
+class CameraParameters(object):
+    camera_matrix = None
+    dist_coefs = None
+    rms = None
```

Comment:
Noob here, what does rms mean?

Reply:
"root mean square". It's the root mean square of the reprojection error. Or something like that. Should be max. 1.0 otherwise the calibration is probably off.

All knowledge taken from here: https://stackoverflow.com/questions/5987285/what-is-an-acceptable-return-value-from-cvcalibratecamera